actionlist/__tests__/components/sideMenuSpec.js

```diff
@@ -13,19 +13,22 @@ jest.unmock('../../app/components/SideMenu');
 describe('SideMenu', () => {
   it('should render the inner tree component structure properly', () => {
     const middlewares = [];
-    const initialState = {};
+    const initialState = {
+      actionItemsReducer: {
```

Comment:
middleware is for the future purposes where we include thunks. Not necessary right now.